src/main/java/com/angelbroking/smartapi/smartstream/models/LTP.java

```diff
@@ -0,0 +1,28 @@
+package com.angelbroking.smartapi.smartstream.models;
+
+import com.angelbroking.smartapi.utils.ByteUtils;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.nio.ByteBuffer;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class LTP {
+	public static final int PACKET_SIZE_IN_BYTES = 51;
+	
+	private TokenID token;
+	private long sequenceNumber;
+	private long exchangeFeedTimeEpochMillis;
+	private long lastTradedPrice;
+
+	public LTP(ByteBuffer buffer) {
+		this.token = ByteUtils.getTokenID(buffer);
+		this.sequenceNumber = buffer.getLong(27);
```

Comment:
27: This integer represents the position in the ByteBuffer from which to read the sequenceNumber value. It indicates that the sequenceNumber value is expected to be found at position 27 in the buffer.

    35: This integer represents the position in the ByteBuffer from which to read the exchangeFeedTimeEpochMillis value. It indicates that the exchangeFeedTimeEpochMillis value is expected to be found at position 35 in the buffer.

    43: This integer represents the position in the ByteBuffer from which to read the lastTradedPrice value. It indicates that the lastTradedPrice value is expected to be found at position 43 in the buffer.
    
    
i have created constant with meaningful name and have used it in LTP.java class instead of directing specifying the position in buffer